uint32 limb
uint32 phaseNumber
uint32 duration
Vector3 translation
Vector3 angle
Vector3 odometryOffset
